onnxruntime.sh

```diff
@@ -5,6 +5,7 @@ source: https://github.com/alisw/onnxruntime.git
 requires:
   - protobuf
   - re2
+  - Python-modules
```

Comment:
Actually, `Python-modules` can be reduced to `Python`, because the intention is to let CMake find the proper Python interpreter. The packages `Python-modules` installs are unnecessary to ONNXRuntime itself.

Reply:
I changed it. Notice you can propose changes directly to the PR by clicking on the +/- sign.